packages/backend/server/src/core/quota/storage.ts

```diff
@@ -98,6 +98,10 @@ export class QuotaManagementService {
     }, 0);
   }
 
+  async getWorkspaceStorageUsage(workspaceId: string) {
```

Comment:
why this wrapper is needed

Reply:
code updated, ensure that size is within the safe range of gql